src/main/java/step4/InputView.java

```diff
@@ -0,0 +1,39 @@
+package step4;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final String HOW_MANY_CAR = "자동차의 대수는 몇 대 인가요?";
+    private static final String HOW_MANY_CYCLE = "시도할 횟수는 몇 회 인가요?";
+    private static final int MIN_ROUND = 0;
+    private static final int MIN_CAR = 0;
+    private static final String CYCLE_ERROR_MESSAGE = "시도할 횟수는 0보다 커야 합니다.";
+    private static final String CAR_ERROR_MESSAGE = "자동차 대수는 1대 이상이어야 합니다.";
+    private Scanner sc = new Scanner(System.in);
```

Comment:
```suggestion
    private static final Scanner SC = new Scanner(System.in);
```
이 부분도 한번만 초기화 될 수 있도록 정적 상수로 선언 되면 좋을 것 같습니다 😄